app/js/controllers.js

```diff
@@ -1797,8 +1797,31 @@ angular.module('myApp.controllers', ['myApp.i18n'])
         })
       }
       if (selectedMessageIDs.length) {
-        ErrorService.confirm({type: 'MESSAGES_DELETE', count: selectedMessageIDs.length}).then(function () {
-          AppMessagesManager.deleteMessages(selectedMessageIDs).then(function () {
+        var peerString = $scope.curDialog.peer
+        var peerID = $scope.curDialog.peerID
+        var firstChar = peerString.charAt(0)
```

Comment:
Okay; I wanted to use those methods but I'm a little confused about the naming. I understand that what you call a megagroup is what I call a supergroup. Is what you call a broadcast what I call a normal group?

Currently, I use these names:

- s: supergroup
- c: channel
- g: (normal) group
- u: user/bot

These are all mutually exclusive (e.g. a channel can never be a supergroup).

Therefore I was quite confused when I read that `isBroadcast(id)` returns `isChannel(id) && !isMegagroup(id)` - if something is a channel, why must we check that it is not a supergroup? Also, `isAnyGroup(peerID)` returns `(peerID < 0) && !AppChatsManager.isBroadcast(-peerID)`. Does this mean that a broadcast is not a group?